_includes/head-custom.html

```diff
@@ -0,0 +1,6 @@
+<!-- start custom head snippets, customize with your own _includes/head-custom.html file -->
+
+<!-- You can set your favicon here -->
+<link rel="shortcut icon" type="image/x-icon" href="{{ '/favicon.ico' | relative_url }}">
```

Comment:
I don't think we need this because it's working without it.  Not sure why 🪄

Reply:
Yeah that's interesting - I think just putting the favicon.ico in place is enough for most browsers. 🤷